content/mtr/mtr.md

```diff
@@ -62,17 +62,17 @@ Commanders are revealed prior to shuffling and presenting decks. A game should n
 
 [Original](https://blogs.magicjudges.org/rules/mtr2-4/)
 
-When time is called, the active player finishes their turn, and there are no additional turns. The game ends when the active player passes their turn. If a non-active player is acting in the end step of the active turn, when time is called, the subsequent turn becomes the final one. A time limit of 20 minutes should be applied to the final turn when time is called.
+When time is called, the active player finishes their turn, and there are no additional turns. The game ends when the active player passes their turn. A time limit of 20 minutes should be applied to the final turn when time is called. If this time limit is reached then the spell or ability that is currently on top of the stack resolves and the rest of the stack is exiled.
```

Comment:
Removal of the extra turn if actions are being taken in the end step probably needs a little discussion too, "actions" is kind of vague and could be clarified to be "casting a spell or activating an ability" but the timing on when you need to be doing that for it to count has caused some confusion with some players. Some question if it counts if they perform actions before time is called if they're in the end step or if it's as time is called or if actions they take after time is called in the end step count. Removing this option helps by just not making this a problem.